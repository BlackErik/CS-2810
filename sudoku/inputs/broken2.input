.......4..3.2.87...2.9.........8........53.9841....2....7...4..34......79..5.7.6.
