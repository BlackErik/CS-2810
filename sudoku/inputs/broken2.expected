Please enter an 81-character board:

Board with up-to-date pencil marks:
+-----------------------+-----------------------+-----------------------+
| 1 . . | . . . | 1 . . | 1 . 3 | 1 . 3 | 1 . . | 1 . 3 |       | 1 2 3 |
| . 5 6   . 5 6   . 5 6 | . . 6   . . 6   . 5 6 | . 5 6     4     . 5 6 |
| 7 8 . | 7 . 9 | . 8 9 | 7 . . | 7 . . | . . . | . 8 9 |       | . . 9 |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
| 1 . . |       | 1 . . |       | 1 . . |       |       | 1 . . | 1 . . |
| . 5 6     3     4 5 6 |   2     4 . 6     8   |   7     . 5 .   . 5 6 |
| . . . |       | . . 9 |       | . . . |       |       | . . . | . . 9 |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
| 1 . . |       | 1 . . |       | 1 . 3 | 1 . . | 1 . 3 | 1 . 3 | 1 . 3 |
| . 5 6     2     4 5 6 |   9     4 . 6   4 5 6 | . 5 6   . 5 .   . 5 6 |
| 7 8 . |       | . 8 . |       | 7 . . | . . . | . 8 . | . 8 . | . . . |
+-----------------------+-----------------------+-----------------------+
| . 2 . | . . . | . 2 3 | 1 . . |       | 1 2 . | 1 . 3 | 1 . 3 | 1 . 3 |
| . 5 6   . 5 6   . 5 6 | 4 . 6     8     4 . 6 | . 5 6   . 5 .   4 5 6 |
| 7 . . | 7 . 9 | . . 9 | 7 . . |       | . . 9 | . . . | 7 . . | . . . |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
| . 2 . | . . . | . 2 . | 1 . . |       |       | 1 . . |       |       |
| . . 6   . . 6   . . 6 | 4 . 6     5       3   | . . 6     9       8   |
| 7 . . | 7 . . | . . . | 7 . . |       |       | . . . |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       | . . 3 | . . . | . . . | . . . |       | . . 3 | . . 3 |
|   4       1     . 5 6 | . . 6   . . 6   . . 6 |   2     . 5 .   . 5 6 |
|       |       | . 8 9 | 7 . . | 7 . 9 | . . 9 |       | 7 . . | . . . |
+-----------------------+-----------------------+-----------------------+
| 1 2 . | . . . |       | 1 . 3 | 1 2 3 | 1 2 . |       | 1 2 3 | 1 2 3 |
| . 5 6   . 5 6     7   | . . 6   . . 6   . . 6 |   4     . 5 .   . 5 . |
| . . . | . . . |       | . 8 . | . . 9 | . . 9 |       | . 8 . | . . 9 |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       | 1 2 . | 1 . . | 1 2 . | 1 2 . | 1 . . | 1 2 . |       |
|   3       4     . 5 6 | . . 6   . . 6   . . 6 | . 5 .   . 5 .     7   |
|       |       | . . . | . 8 . | . . 9 | . . 9 | . 8 9 | . 8 . |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       | 1 2 . |       | 1 2 3 |       | 1 . 3 |       | 1 2 3 |
|   9       8     . . . |   5     4 . .     7   | . . .     6     . . . |
|       |       | . . . |       | . . . |       | . . . |       | . . . |
+-----------------------+-----------------------+-----------------------+

Board after solving naked sets:
+-----------------------+-----------------------+-----------------------+
|       |       |       | . . . |       |       |       |       |       |
|   7       6       9   | . . .     3       5   |   8       4       2   |
|       |       |       | . . . |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       | . . . |       |       | . . . |       |
|   1       3       4   |   2     . . .     8   |   7     . . .     9   |
|       |       |       |       | . . . |       |       | . . . |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       | . . . |       |       |
|   8       2       5   |   9       1       4   | . . .     3       6   |
|       |       |       |       |       |       | . . . |       |       |
+-----------------------+-----------------------+-----------------------+
|       |       |       |       |       |       |       |       |       |
|   5       9       3   |   1       8       2   |   6       7       4   |
|       |       |       |       |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       |       |       |
|   2       7       6   |   4       5       3   |   1       9       8   |
|       |       |       |       |       |       |       |       |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       | . . . |       | . . . |       |       |       |
|   4       1       8   | . . 6     9     . . . |   2       5       3   |
|       |       |       | 7 . . |       | . . . |       |       |       |
+-----------------------+-----------------------+-----------------------+
|       |       |       |       |       |       |       | . . . | . . . |
|   6       5       7   |   3       2       1   |   4     . . .   . . . |
|       |       |       |       |       |       |       | . . . | . . . |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       | . . . |       |       | . . . |       |
|   3       4       1   |   8     . . .     6   |   9     . . .     7   |
|       |       |       |       | . . . |       |       | . . . |       |
| - - - + - - - + - - - | - - - + - - - + - - - | - - - + - - - + - - - |
|       |       |       |       |       |       |       |       |       |
|   9       8       2   |   5       4       7   |   3       6       1   |
|       |       |       |       |       |       |       |       |       |
+-----------------------+-----------------------+-----------------------+

Board is unsolvable
